### DISCUSSION TITLE
formatted-output/formatting

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#535] [Book Comments]
# formatted-output/formatting

Learn Rust with Example, Exercise and real Practice, written with ❤️ by https://course.rs team

https://practice.course.rs/formatted-output/formatting.html

**Comment by TDAkory** [comment]
mark finished

**Comment by 0125nia** [comment]
done

**Comment by DataShare-duo** [comment]
打个卡😄

**Comment by gastrodia** [comment]
deno

**Comment by FlandreLiao** [comment]
Done

**Comment by HenryLunaAdam** [comment]
done.
